pkg/controller/windowsmachineconfig/nodeconfig/nodeconfig.go

```diff
@@ -209,10 +230,9 @@ func (nc *nodeConfig) handleCSR(requestorFilter string) error {
 	return nil
 }
 
-// GetNode returns a pointer to the node object associated with the external IP provided
-func (nc *nodeConfig) getNode(externalIP string) (*v1.Node, error) {
-	var matchedNode *v1.Node
-
+// getNode returns a pointer to the node object associated with the external IP provided
+func (nc *nodeConfig) getNode() (*v1.Node, error) {
```

Comment:
This doesn't need to return anything, you can directly populate nc.node here in this method.